src/components/AccountModule/connection-hooks.js

```diff
@@ -1,28 +1,29 @@
-import { useState, useEffect } from 'react'
```

Comment:
Are the changes in this file just to move the two separate hooks into this file?

Reply:
Yes.